src/components/utils/test/CMakeLists.txt

```diff
@@ -86,9 +87,9 @@ else ()
   # --- Tests for SQLite Wrapper
   find_package(Sqlite3 REQUIRED)
   list (APPEND testSources
-    ./sqlite_wrapper/sql_database_test.cc 
-    ./sqlite_wrapper/sql_query_test.cc   
-    generated_code_with_sqlite_test.cc   
+    ./sqlite_wrapper/sql_database_test.cc
+    ./sqlite_wrapper/sql_query_test.cc
+    generated_code_with_sqlite_test.cc
 
     # TODO{ALeshin} APPLINK-11132 AssertTrue in SetUpTestCase() return false
```

Comment:
Check this commented code. Task is implemented already. If it really ok - uncomment code and delete comment,